src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import Dropdown from './Dropdown';
+import Icon from '../Icon';
+
+export default {
+  title: 'Dropdown',
+  component: Dropdown,
+};
+
+export const DefaultDropdown = () => (
+  <div>
+    <Dropdown onChange={action('selected')} placeholder="select a framework">
```

Comment:
for the action. I would like to know which option I selected in this onChange event.

Reply:
this will print the selected one. You can try it to see ;)